package/MDAnalysis/coordinates/GRO.py

```diff
@@ -244,6 +244,33 @@ def Writer(self, filename, n_atoms=None, **kwargs):
         if n_atoms is None:
             n_atoms = self.n_atoms
         return GROWriter(filename, n_atoms=n_atoms, **kwargs)
+    
+    def add_transformations(self, transform_list):
```

Comment:
GROReader shouldn't be acting like MemoryReader, so this shouldn't be needed.  Ie if GROReader isn't working without this, then there's probably quite a few Readers also not working